source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/FeatureRegistrationRenderTest.java

```diff
@@ -0,0 +1,64 @@
+package io.adaptivecards.objectmodel;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.test.mock.MockContext;
+import android.widget.LinearLayout;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+import io.adaptivecards.renderer.AdaptiveCardRenderer;
+import io.adaptivecards.renderer.RenderedAdaptiveCard;
+import io.adaptivecards.renderer.registration.CardRendererRegistration;
+
+public class FeatureRegistrationRenderTest
+{
+
+    static {
+        System.loadLibrary("adaptivecards-native-lib");
+    }
+
+    @Test
+    public void TestDefaultRendering() throws Exception
+    {
+        final String cardJson  =
+            "{\"$schema\": \"http://adaptivecards.io/schemas/adaptive-card.json\"," +
+                "\"type\": \"AdaptiveCard\"," +
+                "\"version\": \"1.0\"," +
+                "\"body\": [{\"type\": \"TextBlock\",\"text\": \"This is some text\"}]" +
+            "}";
+
+        HostConfig hostConfig = new HostConfig();
+
+        ParseContext parseContext = new ParseContext();
+        ParseResult parseResult = AdaptiveCard.DeserializeFromString(cardJson, AdaptiveCardRenderer.VERSION, parseContext);
+        Context context = InstrumentationRegistry.getTargetContext();
+        AdaptiveCardRenderer.getInstance().render(context, null, parseResult.GetAdaptiveCard(), null, hostConfig);
+    }
+
+    @Test
+    public void TestFeatureRegistrationRendering() throws Exception
```

Comment:
>Exception [](start = 58, length = 9)

Question: What happens if the test does end up throwing ? Does the test harness handle it?